database/factories/JobOfferFactory.php

```diff
@@ -18,8 +19,15 @@ class JobOfferFactory extends Factory
     public function definition(): array
     {
         $faker = \Faker\Factory::create();
```

Comment:
Laravel ya provee una instancia de Faker mediante $this->faker en las clases que extienden de Illuminate\Database\Eloquent\Factories\Factory.

Reply:
Done